app/components/jobseekers/vacancy_summary_component/vacancy_summary_component.html.slim

```diff
@@ -11,8 +11,8 @@ li.vacancy role="listitem" tab-index="0"
     dd.double
       = @vacancy.salary
       - if @vacancy.actual_salary?
-        span.govuk-hint class="govuk-!-margin-bottom-0 govuk-!-margin-top-1 govuk-!-font-size-16" = t("jobs.actual_salary")
-        = @vacancy.actual_salary
+        .govuk-hint class="govuk-!-margin-bottom-0 govuk-!-margin-top-1 govuk-!-font-size-16" = t("jobs.actual_salary")
```

Comment:
it would be more readable if you put
`= t("jobs.actual_salary")`
on a new line